GPU/Common/FramebufferCommon.cpp

```diff
@@ -253,10 +253,10 @@ void FramebufferManagerCommon::EstimateDrawingSize(u32 fb_address, GEBufferForma
 
 	if (viewport_width != region_width) {
 		// The majority of the time, these are equal.  If not, let's check what we know.
-		const u32 fb_normalized_address = fb_address | 0x44000000;
+		const u32 fb_normalized_address = fb_address & 0x3FFFFFFF;
```

Comment:
Hm.  This heuristic probably isn't valid outside VRAM anyway - since we're setting `fb_address` in `GetFramebufferHeuristicInputs`, maybe the simpler thing is just not to mask this at all (removing this line entirely)?

If `fb_address` needs masking, then there's a bug in `GetFramebufferHeuristicInputs`.  `vfbs_[i]->fb_address` possibly still needs masking, of course.

-[Unknown]

Reply:
Yeah, you're right - GetFramebufferHeuristic should only be called when we're actually rendering which means it must be VRAM because the GPU is incapable of rendering ot RAM.

I'll get rid of the unnecessary variable here.